ovgs.proto

```diff
@@ -375,7 +375,8 @@ service OwnershipVoucherService {
   // exists/if applicable)
   // Errors will be returned:
```

Comment:
There's a solid argument that returning Error at the RPC level is just wrong for  pretty much all of these cases.

I'd think that MOST of these are 'business logic' errors, that a proper status returned in the rpc return message would better suit the system.

Reply:
ok.  But, that would be a significant departure from the existing behavior for several deployments.  I think that would need to be a separate effort.